### DISCUSSION TITLE
Solid-Table Examples Do Not Compile

### DISCUSSION THREAD

**Discussion by dabat** [TanStack/table#3933] [Q&A]
Hello,

I am looking into using solid-table and found that the examples do not run. It appears the the API has changed because several elements do not exist in `solid-table`. (e.g. `getCoreRowModelSync, getColumnFilteredRowModelSync, getGlobalFilteredRowModelSync`, etc. there is more)

https://solidjs-table.vercel.app/examples/filtering 

Are there any working examples elsewhere, or will these be updated at some point?

Thanks!

**Comment by tannerlinsley** [comment]
We’ll get these updated asap. There were some renamings. You can remove the Sync prefix for now and it should work 🤞

Tanner Linsley
